R/single_value.R

```diff
@@ -0,0 +1,29 @@
+#' Ensure that a vector has only a single value throughout.
+#'
+#' Missing values are replaced with the single value, and if all values are
+#' missing, the first value in \code{missing} is used throughout.
+#'
+#' @param x The vector which should have a single value
+#' @param missing The vector of values to consider missing in \code{x}
+#' @param warn_if_all_missing Generate a warning if all values are missing?
+#' @param info If more than one value is found, append this to the warning or
+#'   error to assist with determining the location of the issue.
```

Comment:
Can you give an example of how to use `info`?  I see how it works but not when it would be handy.  E.g., if you're using this with `group_by` can it tell you which grouping var has the multiple variables?

Reply:
I added an example.